src/components/CategoryFilter/CategoryFilter.jsx

```diff
@@ -9,16 +15,24 @@ const CategoryFilter = () => {
               type='checkbox'
               name='category'
               className='form-checkbox-field'
-              value='Chuck Taylor Sneakers'
+              value='ChuckTaylor'
+              onChange={(e) =>
+                dispatch({ type: "CATEGORY_FILTER", payload: e.target.value })
+              }
+              checked={ChuckTaylor}
             />
-            Chuck Taylor Sneakers
+            Chuck Taylor
           </label>
           <label className='form-label'>
             <input
               className='form-checkbox-field'
               type='checkbox'
               name='category'
               value='BasketBall'
+              onChange={(e) =>
+                dispatch({ type: "CATEGORY_FILTER", payload: e.target.value })
+              }
+              checked={BasketBall}
             />
             BasketBall
           </label>
```

Comment:
try to make this part a reusable component, so that you can pass your data and just loop over it, instead of repeating yourself